docs/source/configuration/external_timelock_service_configs/timelock_client_config.rst

```diff
@@ -38,12 +40,53 @@ Required parameters:
            Note that client names must be non-empty and consist of only alphanumeric characters, dashes and
            underscores (succinctly, ``[a-zA-Z0-9_-]+``) and for backwards compatibility cannot be the reserved word ``leader``.
 
+Optional parameters:
+
+.. note::
+
+    Specifying the ``serversList`` (a ``ServerListConfig``) in the install configuration has been deprecated, but is
+    maintained for backward compatibility. Please switch to declaring the ``ServerListConfig`` in the runtime
+    configuration as soon as possible.
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Property
+         - Description
+
     *    - serversList::servers
          - A list of all hosts. The hosts must be specified as addresses, i.e. ``https://host:port``.
-           At least one server must be specified. AtlasDB assumes that the Timelock Servers being pointed at
-           are part of the same Timelock cluster.
+           AtlasDB assumes that the Timelock Servers being pointed at are part of the same Timelock cluster.
+           If this is not provided, it defaults to the empty list.
 
-Optional parameters:
+    *    - serversList::sslConfiguration
+         - The SSL configuration of the service. This should follow the
+           `palantir/http-remoting-api <https://github.com/palantir/http-remoting-api/blob/1.4.0/ssl-config/src/main/java/com/palantir/remoting/api/config/ssl/SslConfiguration.java>`__
+           library. This should also be in alignment with the protocol used when configuring the servers.
+
+    *    - serversList::proxyConfiguration
+         - The proxy configuration of the service. This should follow the
+           `palantir/http-remoting-api <https://github.com/palantir/http-remoting-api/blob/1.4.0/service-config/src/main/java/com/palantir/remoting/api/config/service/ProxyConfiguration.java>`__
+           library.
+
+Runtime Configuration
+---------------------
+
+.. warning::
+
+    Although we support starting up without knowledge of any TimeLock nodes, note that if you are using TimeLock
+    your service will fail to start if asynchronous initialization (``initializeAsync``) is set to ``false``, as
+    initializing a ``TransactionManager`` requires communication with TimeLock.
+
+We support live reloading of the ``ServerListConfiguration`` for TimeLock. This can be optionally configured in the
+``timelockRuntime`` block under AtlasDB's runtime configuration root.
+
+Note that if this block is present, then the ``ServerListConfiguration`` in the install configuration will be ignored.
+
+Also, although we support live-reloading of the server configuration, AtlasDB needs to know at install time that it
+should talk to TimeLock - thus, the install configuration must contain a ``timelock`` block (even if said block is
+possibly empty).
```

Comment:
How do I specify an empty block?  Does it contain nothing or does it need something to say it is a object with no fields set?

Reply:
I've added an example of an empty block (it is the YAML empty object: `{}`)